ui/src/plugins/epl-ets/components/TableView.js

```diff
@@ -0,0 +1,285 @@
+// @flow
+import React from 'react';
+
+import AutoSizer from 'react-virtualized/dist/commonjs/AutoSizer';
+import {
+  Table,
+  Column,
+  SortDirection
+} from 'react-virtualized/dist/commonjs/Table';
+import 'react-virtualized/styles.css';
+import './Table.css';
+
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+
+class TableView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sortBy: '',
+      sortDirection: SortDirection.ASC,
+      selectLabel: 'Select columns to show',
+      selectOptions: [
+        { value: 'name', label: 'name' },
+        { value: 'size', label: 'Size' },
+        { value: 'memory', label: 'Memory' },
+        { value: 'type', label: 'Type' },
+        { value: 'write_concurrency', label: 'Write concurrency' },
+        { value: 'read_concurrency', label: 'Read concurrency' },
+        { value: 'lookup_max', label: 'lookup max time' },
+        { value: 'lookup_count', label: 'lookup count' },
+        { value: 'insert_max', label: 'insert max time' },
+        { value: 'insert_count', label: 'insert count' },
+        { value: 'owner', label: 'Owner' },
+        { value: 'heir', label: 'Heir' },
+        { value: 'protection', label: 'Protection' }
+      ],
+      selectedOptions: [
+        { value: 'name', label: 'Name' },
+        { value: 'size', label: 'Size' },
+        { value: 'memory', label: 'Memory' },
+        { value: 'type', label: 'Type' },
+        { value: 'write_concurrency', label: 'Write concurrency' },
+        { value: 'read_concurrency', label: 'Read concurrency' },
+        { value: 'lookup_max', label: 'lookup max time' },
+        { value: 'lookup_count', label: 'lookup count' },
+        { value: 'insert_max', label: 'insert max time' },
+        { value: 'insert_count', label: 'insert count' }
+      ]
+    };
+  }
+
+  is_selected(val) {
+    var listOfObjects = this.state.selectedOptions.filter(({ value, ...r }) => {
+      return value === val;
```

Comment:
Doing simple functions you can simplify function body from statement to expression.
`({value, ...r }) => value ===  val`. In this case you don't need other part of the object you're comparing so you can leave only `({ value })` which will get only what you need from that object.